examples/duet/vae/AutoEncoder_Syft_Data_Owner.ipynb

```diff
@@ -1,8 +1,142 @@
 {
```

Comment:
<p>Can we add the heading cell above this cell like in other examples/duet notebooks: "Data Owner / Data Scientist etc".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5299/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done